creator/users/schema.py

```diff
@@ -180,6 +192,8 @@ def get_node(cls, info, id):
 
 
 class UserFilter(django_filters.FilterSet):
+    organizations = GlobalIDFilter(field_name="organizations")
```

Comment:
i think these should be `organization` now right?

```suggestion
    organization = GlobalIDFilter(field_name="organization")
```

Reply:
true